app/models/concerns/key_pair_compute_resource.rb

```diff
@@ -23,8 +23,8 @@ def recreate
     setup_key_pair
   end
 
-  def delete_key_pair(kay_pair_name)
-    delete_key_from_resource(kay_pair_name)
+  def delete_key_pair(key_pair_name)
+    delete_key_from_resource(key_pair_name)
```

Comment:
I'm not sure why we need this function that only calls another function tbh, but in any case `delete_key_from_resource` no longer takes an argument.